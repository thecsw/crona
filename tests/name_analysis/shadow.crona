a : int;

main : void(a : int, b : int) {
	a = b;
}
