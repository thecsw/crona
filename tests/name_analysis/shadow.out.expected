a(int) : int;
main(int,int->void) : void(a(int) : int, b(int) : int){
	a(int) = b(int);
}